test/pipeline/pipeline.import.azure.master.json

```diff
@@ -524,11 +989,30 @@
                             "definitionType": "task"
                         },
                         "inputs": {
-                            "gitHubConnection": "9d87270a-7fc7-43db-bf57-09f9f4e8879b",
+                            "gitHubConnection": "c381556b-c775-4706-9297-23e2d35b5ac6",
                             "action": "add",
                             "label": "bicep"
                         }
                     },
+                    {
+                        "environment": {},
+                        "enabled": true,
+                        "continueOnError": true,
+                        "alwaysRun": false,
+                        "displayName": "PR label: bicep decompile",
+                        "timeoutInMinutes": 0,
+                        "condition": "and(succeeded(), eq(variables['github.enable.labels'], 'true'), eq(variables['bicep.supported'], 'true'), eq(variables['bicep.decompile'], 'true'), ne(variables['Build.SourceBranch'], 'refs/heads/master'))",
```

Comment:
What's the scenario for this label?  I.e. when/why do we add it?  I thought it was because we didn't want a json only sample to simply be a decompile - ie. contains decompile fluff like var_someName.  (if that's right I wonder if we should rename the label, but probably not worth solving right now

Reply:
"bicep.decompile" -> "decompile clean-up needed"